test/unit/specs/breadcrumb.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { mountWithElement, wait } from '../util'
+
+describe('Breadcrumb', () => {
+  it('create', async () => {
+    const wrapper = mountWithElement({
+      template: `
+      <el-breadcrumb separator=">">
+        <el-breadcrumb-item :to="{ path: '/' }">首页</el-breadcrumb-item>
+        <el-breadcrumb-item><a href="/">活动管理</a></el-breadcrumb-item>
+        <el-breadcrumb-item>活动列表</el-breadcrumb-item>
+        <el-breadcrumb-item>活动详情</el-breadcrumb-item>
+      </el-breadcrumb>
+      `,
+    })
+    await wait()
+    expect(
+      wrapper.find<HTMLElement>('.el-breadcrumb__separator').text()
+    ).toEqual('>')
+  })
+
+  it('seperator-class', async () => {
+    const wrapper = mountWithElement({
+      template: `
+      <el-breadcrumb separator-class="el-icon-check">
+        <el-breadcrumb-item :to="{ path: '/' }">首页</el-breadcrumb-item>
+        <el-breadcrumb-item><a href="/">活动管理</a></el-breadcrumb-item>
+        <el-breadcrumb-item>活动列表</el-breadcrumb-item>
+        <el-breadcrumb-item>活动详情</el-breadcrumb-item>
+      </el-breadcrumb>
+      `,
+    })
+    await wait()
+    expect(
+      wrapper.find<HTMLElement>('.el-breadcrumb__separator.el-icon-check')
+    ).toBeTruthy()
+  })
+})
```

Comment:
Is there an easy way to test with `vue-router` ? Or mock `globalProperties` to test the link click ?

Reply:
Sorry!

After search on web, I can not find a method to mock `globalProperties `.
the `vue-test-utils-next` has a merged pr [https://github.com/vuejs/vue-test-utils-next/pull/83](https://github.com/vuejs/vue-test-utils-next/pull/83) And It's API does has method to mock `globalProperties `

Reply:
I will try to do it later, thanks.